### ISSUE TITLE
install.sh is missing 

### ISSUE THREAD

**Issue by itlinux** [F5Networks/f5-bigip-runtime-init#2]
in terraform sample folder main.tf calls a file install.sh but it's not on the repo, in fact develop/scripts do not exist 

https://raw.githubusercontent.com/f5devcentral/f5-bigip-runtime-init/develop/scripts/install.sh

**Comment by andreykashcheev** [comment]
Thanks for reporting this issue! 

The install script was depreciated and no longer is used for installing BIGIP Runtime Init. Instead of install.sh, self-extracting archive was added to install the tool:
https://github.com/F5Networks/f5-bigip-runtime-init#installer

Self-extracting archive can be found/download from Release page:
https://github.com/F5Networks/f5-bigip-runtime-init/releases/tag/1.0.0

We are going to remove the dead-link. This is track using ESECLDTPLT-2259

Let me know if you have any questions/concerns

**Comment by andreykashcheev** [comment]
Please use the following steps to update example as temporary workaround:

1) Update user_data.txt to have the following:
```
#!/bin/bash

cat << 'EOF' > /config/onboard_config.yaml
runtime_parameters: []
extension_packages:
    install_operations:
        - extensionType: do
          extensionVersion: 1.5.0
        - extensionType: as3
          extensionVersion: 3.13.0
extension_services:
    service_operations: []
EOF

curl -L https://cdn.f5.com/product/cloudsolutions/f5-bigip-runtime-init/v1.0.0/dist/f5-bigip-runtime-init-1.0.0-1.gz.run -o /var/tmp/f5-bigip-runtime-init-1.0.0-1.gz.run
``` 
2) Update main.tf to have the following commandToExecute:
```
    {
      "commandToExecute": "bash /var/tmp/f5-bigip-runtime-init-1.0.0-1.gz.run -- '--cloud azure' 2>&1 && f5-bigip-runtime-init --config-file /config/onboard_config.yaml 2>&1"
    }
```

This would be included with the next release. Let me know if you have any questions/concerns

**Comment by shyawnkarim** [comment]
Closing.

This enhancement was included with [Release 1.1.0](https://github.com/F5Networks/f5-bigip-runtime-init/releases/tag/1.1.0).
